ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,59 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+### Versions
+<!-- Please run `pihole -v` and provide the following: -->
+- Pi-hole:  
+- AdminLTE:  
+- FTL:  
+
+### Platform
+<!-- OS examples: Debian 10, Debian Buster, Ubuntu 18 -->
```

Comment:
Maybe write `Ubuntu 18.04` not sure if everyone will get what `Ubuntu 18` is...

Reply:
Okay, that sounds good.